### PR TITLE
CI refactoring

### PR DESCRIPTION
**Pull request by pi-alexander-popel**
Depends on:
https://github.com/51Degrees/common-ci/pull/27

Also you'll need to rename the master branch to main, because new common-ci setup expects it.

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+    <env name="RESOURCEKEY" value="!!YOUR_RESOURCE_KEY!!"/>
+    <env name="FOD_CLOUD_API_URL" value=""/>
   </php>
   <testsuites>
-    <testsuite name="Tests">
-      <file>examples/index.php</file>
-      <file>tests/geolocation.php</file>
+    <testsuite name="Integration">
+      <file>tests/ExampleTests.php</file>
+      <file>tests/GeoLocationTests.php</file>
```
Comment (ben51degrees) [inline phpunit.xml:9]: Don't `GeoLocationTests` fall into the `Unit` category?

**Review by ben51degrees — CHANGES_REQUESTED** [review]

File: phpunit.xml (hunk 1 above)
Comment (AdamantGarth) [reply phpunit.xml:9]: Not sure, but they require a resource key.

Comment (pi-alexander-popel) [reply phpunit.xml:9]: Not sure, but they require a resource key.

Comment (ben51degrees) [reply phpunit.xml:9]: Fair enough. I suppose if they're calling the actual service, then you're correct, they are integration.

**Review by ben51degrees — APPROVED** [review]

**Pull request merged**